app/src/redux-resources/analytics/hooks/useModuleAnalytics.ts

```diff
@@ -0,0 +1,154 @@
+import { AttachedModule} from '@opentrons/api-client'
+import {
+    useTrackEvent, ANALYTICS_MODULE_COMMAND_ERROR, ANALYTICS_MODULE_COMMAND_COMPLETED
+} from '/app/redux/analytics'
+import { useModulesQuery } from '@opentrons/react-api-client'
+import { ModuleOnlyParams, TemperatureParams } from '@opentrons/shared-data'
+import { Dict } from 'mixpanel-browser'
+import {
+    getAttachedProtocolModuleMatches,
+    getProtocolModulesInfo,
+} from '/app/transformations/analysis'
+import { useNotifyDeckConfigurationQuery } from '/app/resources/deck_configuration'
+import { useMostRecentCompletedAnalysis } from '/app/resources/runs'
+import {
+    FLEX_ROBOT_TYPE,
+    getDeckDefFromRobotType,
+  } from '@opentrons/shared-data'
+const deckDef = getDeckDefFromRobotType(FLEX_ROBOT_TYPE)
+
+// List of commands that will be recorded
+const commandType = [
```

Comment:
```suggestion
const ANALYTIC_COMMAND_TYPES: Array<RunTimeCommand['commandType']> = [
```

Because this is a constant, we generally upper snake case these variables in the app. Also, I highly recommend typing complex strings such as these, because doing so will catch typos for yourself or other devs who extend this data structure (one of these strings does have a typo that typing will catch!).